neurom/fst/tests/test_neuritefunc.py

```diff
@@ -207,6 +207,11 @@ def test_partition():
     assert_allclose(partition,
                     (1.0, 1.0))
 
+def test_partition_asymmetry():
```

Comment:
we should probably create a separate 'SIMPLE' morphology that has more interesting properties than the symmetry we have w/ this one.  Don't need to do it now thought